services/venice-controller/src/main/java/com/linkedin/venice/controller/server/AdminSparkServer.java

```diff
@@ -313,7 +313,7 @@ public boolean startInner() throws Exception {
     httpService.get(CLUSTER_HEALTH_STORES.getPath(), storesRoutes.getAllStoresStatuses(admin));
     httpService.get(STORE.getPath(), storesRoutes.getStore(admin));
     httpService.get(FUTURE_VERSION.getPath(), storesRoutes.getFutureVersion(admin));
-    httpService.get(BACKUP_VERSION.getPath(), storesRoutes.getFutureVersion(admin));
+    httpService.get(BACKUP_VERSION.getPath(), storesRoutes.getBackupVersion(admin));
```

Comment:
Do we have a test for it?

Reply:
yes, I added the test in TestHybrid.java

Reply:
Thanks!